### PR TITLE
Bottle middleware

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,6 +181,23 @@ app.config['PYBRAKE'] = dict(
 app = init_app(app)
 ```
 
+## Bottle integration
+
+Setup airbrake's middleware and project config for your web application:
```
Comment (kyrylo) [inline README.md:186]:
```suggestion
Setup Airbrake's middleware and project config for your web application:
```

**Review by kyrylo — APPROVED** [review]

**Comment by smurf-U** [comment]
PTAL

File: src/pybrake/notifier.py [hunk 2]
```diff
@@ -192,6 +192,7 @@ def _send_notice_sync(self, notice):
             notice["error"] = err
             logger.error(notice["error"])
             return notice
+
```
[the remark this answers is not in the document]
Comment (smurf-U) [reply src/pybrake/notifier.py]: PTAL

[the remark this answers is not in the document]
Comment (kyrylo) [reply src/pybrake/notifier.py]: I already approved this PR. When I approve and have some comments it means I trust you to make the change and merge the PR.

**Review by kyrylo — APPROVED** [review]
Don't forget to squash your temporary commits.

**Pull request merged**